backends/tofino/bf-asm/target.h

```diff
@@ -699,6 +696,11 @@ void emit_parser_registers(const Target::JBay::top_level_regs *regs, std::ostrea
  * will all have a Target::type argument prepended.  The final ARGS argument is the argument
  * list that that will be forwarded (basically ARGDECL without the types)
  */
+#define EXPAND(...) __VA_ARGS__
+#define EXPAND_COMMA(...) __VA_OPT__(, )##__VA_ARGS__                      // NOLINT
+#define EXPAND_COMMA_CLOSE(...) __VA_OPT__(, ) ##__VA_ARGS__ __VA_OPT__()) // NOLINT
```

Comment:
Should the `##` be here?  In addition, the `)` needs to be macro-ized to make paren handling correct.  so should be:

    #define CLOSE_PAREN   )
    #define EXPAND_COMMA_CLOSE(...) __VA_OPT__(, ) __VA_ARGS__ __VA_OPT__(CLOSE_PAREN)

Reply:
It is *probably* ok like this -- `__VA_OPT__()` is a noop no matter how you look at it, and the spurious `##` is probably just ignored, though a strict reading of the spec suggests that having it here is undefined.